### PR TITLE
Added grpc and onnx tests

### PR DESCRIPTION
**Pull request by RajivChitale**
Added tests for MLCompilerBridge gRPC communication.

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -89,4 +95,4 @@ else()
 endif(LLVM_MLBRIDGE)
 
 install(DIRECTORY include/ DESTINATION include)
-install(DIRECTORY CompilerInterface DESTINATION include/python/MLCompilerBridge)
+install(DIRECTORY CompilerInterface DESTINATION ${CMAKE_CURRENT_BINARY_DIR}/MLModelRunner/CompilerInterface)
```
Comment (svkeerthy) [inline CMakeLists.txt]: Please remove ${CMAKE_CURRENT_BINARY_DIR} from here

File: test/mlbridge-test.py [hunk 2]
```diff
@@ -5,22 +5,26 @@
...
 import torch, torch.nn as nn
-
-sys.path.append("../CompilerInterface")
+import subprocess
+import time
+
+BUILD_DIR = "../build_release"
+sys.path.extend(
+    [
+        "../CompilerInterface",
```
Comment (svkeerthy) [inline test/mlbridge-test.py]: Please use CompilerInterface from the build directory

**Review by svkeerthy — CHANGES_REQUESTED** [review]

File: test/include/HelloMLBridge_Env.h [hunk 3]
```diff
@@ -0,0 +1,40 @@
+//===----------------------------------------------------------------------===//
```
Comment (svkeerthy) [inline test/include/HelloMLBridge_Env.h]: Please change the header text. This seems to be the  old license.

File: test/include/ProtosInclude.h
Comment (svkeerthy) [inline test/include/ProtosInclude.h:1]: Please add header with license text.

**Review by svkeerthy — CHANGES_REQUESTED** [review]

File: test/include/HelloMLBridge_Env.h (hunk 3 above)
Comment (RajivChitale) [reply test/include/HelloMLBridge_Env.h]: Changed it to the new license.

**Review by svkeerthy — APPROVED** [review]

**Pull request merged**